django_mock_queries/query.py

```diff
@@ -267,16 +267,22 @@ def update(self, **attrs):
         return count
 
     def _delete_recursive(self, *items_to_remove, **attrs):
+        removed_items = defaultdict(int)
+
         for item in matches(*items_to_remove, **attrs):
             self.items.remove(item)
             self.fire(item, self.EVENT_DELETED)
+            item_label = item._meta.label if hasattr(item, '_meta') else type(item).__name__
```

Comment:
I had to come up with this alternative labelling for, essentially, non-`MockModel` items because I noticed some unit tests were declaring `MockSet` out of native values.

Example:

```python
items = [1, 2, 3]
assert MockSet(*items).count() == len(items)
```

Reply:
If I am not mistaken, we do not have a test for the return value in this case? Could you please add one?

Reply:
I'll add it, but I'm not too sure of its value. Is this something we should support even?

Reply:
Migrating to a label-only approach, id est avoiding the fallback, might be a breaking change if I understand it correctly. But I am unsure about the actual value used here - would `None` be enough or not?

Reply:
> might be a breaking change if I understand it correctly

Technically, yes.

But the intended use case is not really documented anywhere. I only came to the realisation that this was a problem after I saw a `MockSet` of `int` in the tests. IRL, this is not something Django supports. In fact if you try to create a `QuerySet` of anything that doesn't have a `_meta` attribute, it blows up:

```python
from django.db.models import QuerySet

QuerySet(1)
# AttributeError: 'int' object has no attribute '_meta'
```